cmd/skopeo/inspect.go

```diff
@@ -22,6 +23,7 @@ import (
 type inspectOptions struct {
 	global    *globalOptions
 	image     *imageOptions
+	format    string
 	retryOpts *retry.RetryOptions
```

Comment:
(Absolutely non-blocking nit: move the `format` option one line down, below the “use shared options” sets.)

Reply:
Not sure what you mean here?

Reply:
```go
type inspectOptions struct {
	global    *globalOptions
	image     *imageOptions
	retryOpts *retry.RetryOptions
	format    string
```

Reply:
Ok, but why?

Reply:
BTW I did it, but I am curious why you want this moved?

Reply:
The `*Options` values are conceptually different (sets of options shared between commands) from the other command-specific options, so it seems more intuitive for me to group the like together.

Also, that allows the declaration of the struct and its preparation for use in `inspectCmd` to be roughly in the same order, making it easier to verify that the two correspond.

I did say it is an “absolutely non-blocking nit”.